### ISSUE TITLE
Drop cask with Spacemacs icon?

### ISSUE THREAD

**Issue by railwaycat** [railwaycat/homebrew-emacsmacport#288]
Not sure if I should still keep build a separate cask for Spacemacs icon.

**Comment by WannabeSmith** [comment]
It might be nice to have an emacs icon that matches the new icons of macOS 11+. For example, this was discussed in an [issue for emacs-plus ](https://github.com/d12frosted/homebrew-emacs-plus/issues/303#issue-782591387) where elrumo provided some [really nice icons](https://github.com/d12frosted/homebrew-emacs-plus/issues/303#issuecomment-763928162).

**Comment by jmhammond** [comment]
I put together [this icon](https://github.com/jmhammond/emacs-icon/) that I've been enjoying, in case anyone else is interested.

**Comment by sankalp-khare** [comment]
I use the spacemacs cask 😢

Is it very troublesome to maintain? (Trying to understand reason for wanting to drop)

**Comment by WannabeSmith** [comment]
@railwaycat gotcha! Formulae are a little more cumbersome for new users (compilation + discovering that you can't open it from spotlight until properly linked, etc.) so in my mind it makes sense to have the macOS 11+ icon as the default. However, I do understand that it's not the official icon provided by emacs. Just my 2c. 

Also, thanks for making the mituharu port easily installable on macOS!

**Comment by railwaycat** [comment]
We will keep spacemacs icon:)

BTW A friend of mine showed me a tip to replace icon of an app bundle without break the signature. I will add a wiki page or readme section for it.
